presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -92,6 +94,7 @@ public void setup()
                 "\"col_7\" TINYINT, " +
                 "\"col_8\" SMALLINT, " +
                 "\"col_9\" INTEGER, " +
+                "\"col_10\" CHAR, " +
```

Comment:
This should actually be something like `CHAR(50)` since CHAR is fix-width typed that is space-padded to always be the declared width.

(it's a weird concept ... it seems to be a holdover from mainframe-era systems that used fix-width flat file databases)